utils/notebook_version_standardizer.py

```diff
@@ -0,0 +1,149 @@
+import click
+import json
+import os
+
+DOCS_PATH = os.path.join(os.path.dirname(os.path.abspath(__file__)), "source")
```

Comment:
There is no `source` directory in this repo, so we will need to update this path

Reply:
Decided to remove the notebook standizer